README.md

```diff
@@ -12,6 +12,7 @@ distributions.</p>
     sudo snap install wire
     snap connect wire:camera core:camera
     snap connect wire:mount-observe core:mount-observe
+    snap connect wire:password-manager-service
```

Comment:
This seems to be enough, I'll be happy to have some input on whether or not this is correct.

I ran into [an issue](https://github.com/snapcrafters/wire/issues/15#issuecomment-2060975073) with my locally built Snap the first time on a fresh Ubuntu 23.10 install with no update, but subsequent builds worked on up to date Ubuntu 23.10 and Kubuntu 23,10